dockerfiles/launcher/launcher_funcs.sh

```diff
@@ -166,6 +166,22 @@ docker_run_with_debug() {
   fi
 }
 
+docker_run_with_volumes_lib() {
+  if has_lib_volume; then
+    docker_run_with_debug -v "${CHE_LIB_VOLUME}":/data/lib "$@"
```

Comment:
Ok - you will need to wait for the new Che CLI to be merged and then we need to refactor what you are doing with the che-launcher, as the che-launcher will be no more in favor of a newer solution.  

https://github.com/eclipse/che/pull/3301

Am I understanding you correctly in that the eclipse/che-server image will have two requirements:
1. <local-path>:/data/lib to the eclipse/che-server 'docker run' command?
2. AND, che.workspace.volumes_from= must be set within the che.properties in the che-server?

Are both required?  If so, in reading the description, it is a bit unclear for me what each part is providing. could you just explain that.  And when you do that, I can give some pointers on how to refactor the new CLI to support this.